desktop/package/package.gradle

```diff
@@ -7,40 +7,40 @@ task jpackageSanityChecks {
     description 'Interactive sanity checks on the version of the code that will be packaged'
 
     doLast {
-        if (!System.getenv("CI")) {
-            executeCmd("git --no-pager log -5 --oneline")
-            ant.input(message: "Above you see the current HEAD and its recent history.\n" +
-                    "Is this the right commit for packaging? (y=continue, n=abort)",
-                    addproperty: "sanity-check-1",
-                    validargs: "y,n")
+        if (!System.getenv('CI')) {
```

Comment:
Please undo all changes from `"` to `'`. If preferred, it's better to open a PR specifically for that (or at least a separate commit).